### DISCUSSION TITLE
[grpc] Support secure grace but ignore problems

### DISCUSSION THREAD

**Discussion by gAmUssA** [Kong/insomnia#3796] [Ideas]
grpcurl has `-insecure` flag that allows ignoring validation. it would be great to have it in insomnia.

`grpcurl -insecure 34.69.7.159:443 io.kong.developer.quoteservice.QuoteService/GetQuote`

**Comment by filfreire** [comment]
Hi @gAmUssA, we have an internal issue to hopefully look into to this. We'll update here if there's any changes around this introduced in one of the upcoming beta builds.

https://linear.app/insomnia/issue/INS-1547/support-insecure-connection-setup-on-grpc-requests

**Comment by KimTaehee** [comment]
Hello. Was the issue resolved? Then, which version can I request using the insecure option?
